LayoutTests/fast/scrolling/mac/scrollbars/scrollbar-width.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html> <!-- webkit-test-runner [ MockScrollbarsEnabled=false AsyncOverflowScrollingEnabled=true ] -->
```

Comment:
Do we need more tests, e.g. for iframes and for iOS?

Can the containing page style the scrollbars of an iframe? (Currently we do allow iframes to get the `-::webkit-scrollbr` styles from their enclosing frame, even cross-origin, which we should probably kill.)

Reply:
I'll add another test for iframes. iOS will be a follow up patch. The spec doesn't make any mention of a containing page being able to style the scrollbars of an iframe, but I could see that being fairly useful. Not sure if we want to allow it though.